fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -74,18 +73,19 @@ Usage: ${script_name} [options]
   --fmriname=<name of fmri run> (only needed for SEBASED bias correction method)
   --subjectfolder=<subject processing folder> (only needed for TOPUP distortion correction method)
 
-  --biascorrection=<method of bias correction>
+  --biascorrection=<method to use for receive coil bias field correction>
 
         "SEBASED"
-             use bias field derived from spin echo, must also use --method=${SPIN_ECHO_METHOD_OPT}
+             use bias field derived from spin echo images, must also use --method=${SPIN_ECHO_METHOD_OPT}
 
         "LEGACY"
-             use the bias field derived from T1w and T2w images, same as pipeline version 3.14.1 or older"
+             use the bias field derived from T1w and T2w images, same as was used in
+             pipeline version 3.14.1 or older. No longer recommended.
 
         "NONE"
              don't do bias correction
 
-  --usejacobian=<"true" or "false">
+  --usejacobian=<TRUE or FALSE>
```

Comment:
`--use-jacobian` does not currently accept anything other than all-lowercase, this usage change is currently wrong.

Reply:
Good catch. I could add the same case conversion logic that we have in `GenericfMRIVolume`.  Or just revert the Usage statement to the lower-case version.  Which do you prefer?

Reply:
It is an internal script, it doesn't need as much user-friendliness, but it can't hurt.

Reply:
Done.  Forced to lower-case internally within the script, but left as upper case in the Usage statement to match our "general" boolean style.  (Although we aren't entirely consistent about this -- e.g., `--preregistertool` and `--fmrirefreg` are specified in lower case (and lower case only).